CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 
+# 2.12.0 
+
+## Added suport for PowerShell Core
+
+* Added: `Get-CCimInstance` function that determines which method to call when accessing a computer's Common Information Model based on the version of PowerShell being used. `Get-CCimInstance` will call `Get-CimInstance` under PowerShell Core and `Get-WmiObject` under Windows PowerShell. `Get-WmiObject` has been deprecated under PowerShell Core.
+* Fixed: Carbon will now use version 2 of the `PSDesiredStateConfiguration` module when running under PowerShell Core as verion 1 was incompatible.

```

Comment:
Remove this. We only use PSDesiredStateConfiguration in our tests. Nothing that Carbon ships with uses or requires it.